Source/WebCore/platform/graphics/mac/controls/ControlMac.h

```diff
@@ -63,8 +63,11 @@ class ControlMac : public PlatformControl {
 
     void updateCellStates(const FloatRect&, const ControlStyle&) override;
 
-    void drawCell(GraphicsContext&, const FloatRect&, float deviceScaleFactor, const ControlStyle&, NSCell *, NSView *, bool drawCell = true);
-    void drawCellOrFocusRing(GraphicsContext&, const FloatRect&, float deviceScaleFactor, const ControlStyle&, NSCell *, NSView *, bool drawCell = true);
+    void drawCellInternal(const FloatRect&, float deviceScaleFactor, const ControlStyle&, NSCell *);
+    void drawCellFocusRingInternal(const FloatRect&, float deviceScaleFactor, const ControlStyle&, NSCell *);
```

Comment:
Can these "Internal" methods, be `private` rather than `protected`?

Reply:
Fixed. And I made `TextFieldMac::draw()` call `drawCell()` instead of calling `drawCellInternal()`.